components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint/src/main/webapp/password-recovery-notify.jsp

```diff
@@ -114,9 +115,9 @@
             <%
             try {
             %>
-                location.href = "<%= IdentityManagementEndpointUtil.getURLEncodedCallback(callback)%>";
+                location.href = "<%= IdentityManagementEndpointUtil.encodeURL(callback)%>";
```

Comment:
If "getURLEncodedCallback" has issues, check and update it's usages to the new method, and deprecate that

Reply:
Done